pkg/mic/mic.go

```diff
@@ -118,7 +119,7 @@ func NewMICClient(cloudconfig string, config *rest.Config, isNamespaced bool, sy
 
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: clientSet.CoreV1().Events("")})
-	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: aadpodid.CRDGroup})
+	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: aadpodv1.CRDGroup})
```

Comment:
I think we should use recorderv1 to use for cases where the v1 is used to distinguish the code. Thoughts ?

Reply:
Updated this to just use the internal API, as the CRDGroup is just a static string.